bsp_diff/celadon_ivi/kernel/linux-intel-lts2021/0002-Introduce-v4l2loopback-implementation.patch

```diff
@@ -0,0 +1,8271 @@
+From 1a0c34e86bdb0b05a08668fd41720641f22a22f1 Mon Sep 17 00:00:00 2001
+From: Neo Fang <neo.fang@intel.com>
```

Comment:
please split this into 2 patch files. 1st patch file only upstream code. 2nd patch file our changes on top of upstream patch file.

Also generate the patch file using "git format-patch --no-numbered -n" where n is the number of patches that needs to be created and copied to utils folder